### PR TITLE
CRYP-110: Setup BtcEdge API

### PR DESCRIPTION
**Pull request by jason-gerard**
JIRA: https://cryptify.atlassian.net/browse/CRYP-110

## Changes
- Add BtcEdge service
- Create new docker file
- Update dockercompose file
- Update sonarqube

### CODE REVIEW

File: sonarqube-scanner.js [hunk 1]
```diff
@@ -3,11 +3,11 @@ scanner({
...
     options: {
-        "sonar.sources": "./packages/api/src, ./packages/eth-edge/src, ./packages/client",
-        "sonar.tests": "./packages/api/test, ./packages/eth-edge/test",
+        "sonar.sources": "./packages/api/src, ./packages/eth-edge/src, ./packages/btc-edge/src, ./packages/client",
+        "sonar.tests": "./packages/api/test, ./packages/eth-edge/test, ./packages/btc-edge/test",
         "sonar.test.inclusions": "**/*.spec.ts",
         "sonar.typescript.lcov.reportPaths": "**/coverage/lcov.info",
         "sonar.exclusions": "**/android/**/*, **/ios/**/*, **/test/**/*, **/tests/**/*, **/icons/**/*",
-        "sonar.testExecutionReportPaths": "./packages/api/test-report.xml, ./packages/eth-edge/test-report.xml"
+        "sonar.testExecutionReportPaths": "./packages/api/test-report.xml, ./packages/eth-edge/test-report.xml, ./packages/eth-edge/test-report.xml"
```
Comment (alexbara2000) [inline sonarqube-scanner.js:11]: change to btc-edge

**Review by alexbara2000 — APPROVED** [review]
Looks good, all testing and metrics have been taken into consideration

**Review by domsec — APPROVED** [review]
Looks good

File: sonarqube-scanner.js (hunk 1 above)
Comment (jason-gerard) [reply sonarqube-scanner.js:11]: oups

**Pull request merged**